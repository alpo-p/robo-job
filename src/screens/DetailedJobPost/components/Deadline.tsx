import React from 'react'
import { View } from 'react-native'
import { Ionicons } from '@expo/vector-icons'
import { useTheme } from '@react-navigation/native'
import RoboText from '../../../common/components/RoboText'

export const Deadline: React.FC<{ dl: string }> = ({ dl }) => {
  const { colors } = useTheme()
  return (
    <View
      style={{
        flexDirection: 'row',
      }}
    >
      <Ionicons
        style={{ marginTop: -3, marginRight: 2 }}
        name="timer-outline"
        color={colors.text}
        size={20}
      />
      <RoboText weight="light" size="small">
        Deadline: {dl}
      </RoboText>
    </View>
  )
}
