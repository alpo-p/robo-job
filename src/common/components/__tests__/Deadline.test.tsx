import React from 'react'
import rendererCreateWithNavigation from '../../../testUtils/rendererCreateWithNavigation'
import { Deadline } from '../Deadline'

describe('CompanyLogo', () => {
  it('should match snapshot', () => {
    const tree = rendererCreateWithNavigation(<Deadline dl="1" />)
    expect(tree).toMatchSnapshot()
  })
})
