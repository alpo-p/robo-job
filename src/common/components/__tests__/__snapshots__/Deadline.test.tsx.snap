// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompanyLogo should match snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <Text />
  <Text
    style={
      Object {
        "color": "#000000",
        "fontFamily": "DamascusLight",
        "fontSize": 14,
        "textTransform": "none",
      }
    }
  >
    Deadline in 
    1
     days
  </Text>
</View>
`;
